org.alienarena.alienarena.metainfo.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.alienarena.alienarena</id>
+  <launchable type="desktop-id">alien-arena.desktop</launchable>
```

Comment:
this value is wrong. it MUST match the name of the desktop file. That's why the build fails.